packages/api/src/external/commonwell/patient.ts

```diff
@@ -292,6 +292,10 @@ export async function update(
       createContext,
       getOrgIdExcludeList
     );
+
+    // Update the scheduled query request ID if it exists
+    // and retrigger the scheduled query
```

Comment:
Let's avoid comments to explain what it does - that's a sign its not clear/polished yet. Comments should mostly explain "why", not "how" - the code should be clear to understand the "how" - [reference](https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#fb560de23853473f88ea7011630eb6b0).